setup.py

```diff
@@ -121,6 +121,9 @@
             "langchain",
             "openai",
         ],
+        "motherduck": [
+            "duckdb==0.9.2",
```

Comment:
I think we could safely say `duckdb<0.10.0` to make sure we get bug fixes until they support ` `0.10.0`. It would be strange for them to release a `0.9.3` and then not support that bugfix in motherduck.